onadata/libs/utils/export_builder.py

```diff
@@ -911,7 +934,12 @@ def _is_numeric(xpath, element_type, data_dictionary):
             [(_var_types[item],
                 0 if item in ['_id', '_index', '_parent_index',
                               SUBMISSION_TIME] else 255)
-                for item in self.EXTRA_FIELDS]
+                for item in self.EXTRA_FIELDS] +
+            [(x[1],
+              0 if _is_numeric(x[0],
+                               get_element_type_from_xpath(x[0], elements),
+                               self.dd) else 255)
```

Comment:
can we do a find and search to replace `255` with a constant (under the no-magic-numbers justification)? I see at least once place right above here, line 936, seems reasonable to call this `MAX_TEXT_LENGTH` if that's what this denotes